src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -283,7 +283,8 @@ void RPCHandlerImpl::GetMessageVersion(
 bool RPCHandlerImpl::ConvertMessageToSO(
     const Message& message,
     ns_smart_device_link::ns_smart_objects::SmartObject& output,
-    const bool remove_unknown_parameters) {
+    const bool remove_unknown_parameters,
+    const bool validate_existing_params) {
```

Comment:
I think that `validate` would be descriptive enough alone

Reply:
Eh, don't want to confuse it with the validate function(in terms of readability). Could rename it to `validate_params` or `validate_rpc`

Reply:
Ah, got you. I think `validate_params` would be fine